lib/signaling/v2/peerconnection.js

```diff
@@ -323,7 +325,7 @@ class PeerConnectionV2 extends StateMachine {
           || preferredCodecs.audio.some(({ codec, dtx }) => codec === 'opus' && dtx)
       },
       _shouldApplySimulcast: {
-        value: (isChrome || isSafari) && preferredCodecs.video.some(
+        value: (isChrome || isSafari || isChromeIOS) && preferredCodecs.video.some(
```

Comment:
I think `isChrome` will cover iOS chrome as well here:
```suggestion
        value: (isChrome || isSafari) && preferredCodecs.video.some(
```